app/controllers/discourse_post_event/invitees_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
       event_invitees = event.invitees
       event_invitees = event_invitees.with_status(params[:type].to_sym) if params[:type]
 
-      possible_invitees = []
+      suggested_users = []
```

Comment:
This is recent code sam added, I just changed the name as we actually return users and not invitees